task_reactredux/task1/src/containers/CounterAddDelContainer.js

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import CounterContainer from './CounterContainer';
+import AddDelCounter from '../views/AddDelCounter/index';
+
+class CounterAddDelContainer extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            counterArr: [<CounterContainer  />],
+            command: "start"
+        };
+        
+        this.onAddBtnClick = this.onAddBtnClick.bind(this);
+        this.onDelBtnClick = this.onDelBtnClick.bind(this);
+        this.onDefBtnClick = this.onDefBtnClick.bind(this);
+    }
+
+    onAddBtnClick() {
+        let counter = this.state.counterArr;
+
+        counter.push(<CounterContainer />);
+        this.setState({
+            command: "add",
+            counterArr: counter
+        });
```

Comment:
When you are changing state, you should change the reference to object/array in the state. In your code, you changed array, but the reference to it is still the same. It's wrong. You could use the spread operator(`...`) to create a copy of the old array and push a new array element into it.